src/targets/gpu/target.cpp

```diff
@@ -121,7 +121,7 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
         inline_module{},
         rewrite_pooling{},
         dead_code_elimination{},
-        rewrite_gelu{},
+        enable_pass(options.fast_math, rewrite_gelu{}),
```

Comment:
It might be better to add its own flag so it can be independent from the other fast math optimizations. We could just add `MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_DISABLE_FAST_GELU)` 
at line 77 and make this `enable_pass(not enabled(MIGRAPHX_DISABLE_FAST_GELU{}), rewrite_gelu{}),` Then it would also be a run time option instead of a compile time option.

Reply:
Yes, I thought of ENV flag as well. 

But then we have many Env flags around eg. fast_softmax, convert_instancenorm and `disable_fast_gelu` etc. 

Also i found that we are not even using the `fast_math` compile option anywhere.  We may want to unify later all env flag under `fast_math` probably.

Reply:
Made it a env flag for now.  we can discuss more on this when paul's back.

Reply:
This should just use the `fast_math` flag. The reason we dont use that for fast softmax is because it can easily lead to nans rather than less accurate results.